responsive-web-design/Build-product-landing-page/creative.css

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    text-decoration: none;
+}
+
+body {
+    background-image: linear-gradient( 115deg, #ed4b4bba, #285da2a6, #834343), url(backgroundimg.jpg);
```

Comment:
Please use a descriptive name for the background image, for example: `homepage_background.jpg`